packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingTransactions.tsx

```diff
@@ -41,6 +43,11 @@ export const PendingTransactions = ({
     ? 'bg-gray-dark'
     : 'bg-ocl-blue'
 
+  // Show from 13th August 2023
+  const showSubgraphMaintenanceMessage = dayjs()
```

Comment:
No end of time window check?

Reply:
i supposed we can manually delete it easily

Reply:
If we have no end time we can't forget to remove it 😄